CMakeLists.txt

```diff
@@ -60,13 +60,14 @@ endif()
 if(EMSCRIPTEN)
   message("Building for Emscripten")
   set(MANIFOLD_FLAGS -fexceptions -D_LIBCUDACXX_HAS_THREAD_API_EXTERNAL -D_LIBCUDACXX_HAS_THREAD_API_CUDA)
-  # ALLOW_MEMORY_GROWTH is slow, so we just allow 2gb of memory usage
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -sMODULARIZE=1 -sEXPORT_ES6=1 -sINITIAL_MEMORY=2gb -fexceptions -sDISABLE_EXCEPTION_CATCHING=0")
```

Comment:
Did you intend to remove MODULARIZE and EXPORT_ES6?  Or are they already set somewhere else?

Reply:
yes, they are already set for the js binding